### ISSUE TITLE
Backticks Cause Characters to Render to the Current File

### ISSUE THREAD

**Issue by Jiayou-Chao** [continuedev/continue#3503]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Windows 11
- Continue version:0.9.246
- IDE version: VSCode 1.96.2
```

### Description

It seems that any single character enclosed in backticks (\`) will be rendered to the current file in the output. For example, below is the raw output of the LLM:

```
`a` and `b`
```
![image](https://github.com/user-attachments/assets/0041077c-d428-4ef9-8469-eb16f8cd26db)
As can be seen, both `a` and `b` are rendered into the context file.

### To reproduce

Use any text file as the context and ask 
```
Ignore the script. Output "`a ` and `b`" (including the backticks)
```

**Comment by sestinj** [comment]
Good catch. I think we'll need to take a second look at this line here: https://github.com/continuedev/continue/blob/a0eb5bba93933dd112f9e38b1310d8ddca92b7b1/gui/src/components/markdown/StyledMarkdownPreview.tsx#L302

**Comment by RomneyDa** [comment]
Fixed by https://github.com/continuedev/continue/pull/3624
